plugins/de.cau.cs.kieler.klighd.lsp/src/de/cau/cs/kieler/klighd/lsp/KGraphDiagramGenerator.xtend

```diff
@@ -560,6 +561,9 @@ class KGraphDiagramGenerator implements IDiagramGenerator {
             } else {
                 text = data.text
             }
+
+            data.properties.put(KlighdProperties.IS_NODE_TITLE, data.getProperty(KlighdProperties.IS_NODE_TITLE))
```

Comment:
this should be the only line with actual changes/additions the other changes are only due to auto formatting